### PR TITLE
OpenCV and OpenCV Manager version++

### PR DESCRIPTION
**Pull request by asmorkalov**

### CODE REVIEW

**Comment by asmorkalov** [comment]
@jet47 please look at it.

**Comment by jet47** [comment]
:+1:

File: doc/tutorials/introduction/android_binary_package/O4A_SDK.rst [hunk 1]
```diff
@@ -157,10 +157,10 @@ Get the OpenCV4Android SDK
 
    .. code-block:: bash
 
-      unzip ~/Downloads/OpenCV-2.4.10-android-sdk.zip
+      unzip ~/Downloads/OpenCV-2.4.11-android-sdk.zip
 
-.. |opencv_android_bin_pack| replace:: :file:`OpenCV-2.4.10-android-sdk.zip`
-.. _opencv_android_bin_pack_url: http://sourceforge.net/projects/opencvlibrary/files/opencv-android/2.4.10/OpenCV-2.4.10-android-sdk.zip/download
+.. |opencv_android_bin_pack| replace:: :file:`OpenCV-2.4.11-android-sdk.zip`
+.. _opencv_android_bin_pack_url: http://sourceforge.net/projects/opencvlibrary/files/opencv-android/2.4.10/OpenCV-2.4.11-android-sdk.zip/download
```
Comment (Dikay900) [inline doc/tutorials/introduction/android_binary_package/O4A_SDK.rst]: shouldn't it be:
[...]opencv-android/**2.4.11**/OpenCV-2.4.11-android-sdk.zip/download

Comment (asmorkalov) [inline doc/tutorials/introduction/android_binary_package/O4A_SDK.rst]: Definetely! Thank you!

**Comment by Dikay900** [comment]
:+1:

**Pull request merged**